### DISCUSSION TITLE
Confused about UTM params

### DISCUSSION THREAD

**Discussion by tiagoapp** [plausible/analytics#2863] [Anything Else]
Hi,

I'm quite confused about how the UTM params are tracked.
I've Plausible running on a an active project with campaigns running and the numbers don't seem to add up.

For example:
domain.com/?utm_campaign=social&utm_content=QR:123 
domain.com/?utm_campaign=digital&utm_content=QR:456

For example I seem to have a good amount of UTM Campaign 'social' listed on the dashboard. 
However, very little items from digital, but the QR code scanner says the digital is actually the most scanned source.

I've changed my browsers, clear history and change ip / VPN and tried something like:
domain.com/?utm_campaign=test&utm_content=QR:test

And it doesn't registers at all.

So I'm confused in understanding why some are registering and some aren't or what's the principle behind this?

**Comment by metmarkosaric** [comment]
thanks for reaching out @tiagoapp! from looking at the link examples you shared, you don't seem to use utm_source. utm_source is a required parameter and the tracking for utm_campaign etc doesn't work without also having utm_source in the link

**Reply by tiagoapp** [reply]
Thanks @metmarkosaric - actually ended up figuring out my problem. Using the node package on a SPA I've not noticed I needed to append `window.location.search` on the pageview tracking. Eventually come across that note around your documentation. After doing so, every UTM is getting tracked. Just strange some of them were working to begin with!
